docs/book/src/migration/plugin/v2_v3.md

```diff
@@ -1,3 +1,307 @@
-# Migration from `go.kubebuilder.io` v2 to v3
+# Migrating your projects to use go/v3 plugin
 
-TODO
+Make sure you understand the [differences between Kubebuilder v2 and v3](/migration/v2vsv3.md)
+before continuing.
+
+Please ensure you have followed the [installation guide](/quick-start.md#installation)
+to install the required components.
+
+<aside class="note attention">
+<h1>Note</h1>
+
+This guide ONLY describe the steps required to change the layout of your project to starts to use the plugin `go/v3` but will not help you to address all bug fixes in improvements done in its scaffolds for what is generated already.
+
+In this way, it is recommended to upgrade your project by following the [Migration Guide v2 to V3][migration-guide-v2-to-v3] instead which is the most assertive and straightforward way.
+</aside>
+
+## Steps to migrate
+
+<aside class="note attention">
```

Comment:
`attention` isn't a valid class here, iirc (and this should have a proper title besides just `note`)